ruff_cli/src/cli.rs

```diff
@@ -180,7 +180,7 @@ pub struct Cli {
         conflicts_with = "stdin_filename",
         conflicts_with = "watch",
     )]
-    pub explain: Option<RuleCode>,
+    pub explain: Option<String>,
```

Comment:
Maybe the one "loss" in this series of commits. I wish we could catch these statically as part of the CLI.

Separately, while we're here (and now that the type is string), can we set `value_name = "RULE_CODE"` for this?

Reply:
I have changed this to `Option<Rule>` by setting the clap option `value_parser=Rule::from_code`.

Reply:
>can we set value_name = "RULE_CODE" for this?

I agree that this would be a good addition however this PR accomplishes one thing and I'd rather keep unrelated changes for other PRs.

Reply:
All good, preserving the type is great.